subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/NebulaPluginsSmokeTest.groovy

```diff
@@ -75,13 +75,6 @@ class NebulaPluginsSmokeTest extends AbstractPluginValidatingSmokeTest implement
 
         then:
         runner('groovydoc')
-            .expectDeprecationWarning(
```

Comment:
How can this removal work in isolation? Don't we need to update the plugin version?

Reply:
See line 55; the test is ignored because the plugin is already incompatible with Gradle 8.

Reply:
Duh, I was even the one disabling it ... sorry